classes/privacy/provider.php

```diff
@@ -0,0 +1,33 @@
+<?php

```

Comment:
line endings are windows ('\r\n'), should be Unix/MacOS ('\n') to meet Moodle standards